src/main/java/org/opensrp/web/security/DrishtiAuthenticationProvider.java

```diff
@@ -100,15 +99,13 @@ public boolean supports(Class<?> authentication) {
 	}
 
 	protected List<SimpleGrantedAuthority> getRolesAsAuthorities(User user) {
-		return Lambda.convert(user.getRoles(), new Converter<String, SimpleGrantedAuthority>() {
-
-			@Override
-			public SimpleGrantedAuthority convert(String role) {
-				if (GET_ALL_EVENTS_ROlE.equals(role))
-					return new SimpleGrantedAuthority("ROLE_ALL_EVENTS");
-				return new SimpleGrantedAuthority("ROLE_OPENMRS");
-			}
-		});
+		List<SimpleGrantedAuthority> simpleGrantedAuthorities = new ArrayList<>();
+		if(user != null && user.getRoles()!=null) {
```

Comment:
does not look like both are performing same function... if (GET_ALL_EVENTS_ROlE.equals(role))...

Reply:
plz ignore this